maven-plugin/src/main/java/org/openmrs/maven/plugins/Run.java

```diff
@@ -115,6 +123,34 @@ public void executeTask() throws MojoExecutionException, MojoFailureException {
 		}
 	}
 
+	private void validatePort() {
+		boolean inUse = serverHelper.isPortInUse(port);
+		do {
+			if (inUse) {
+				String message = String.format("Port %s is already in use. Can we use %s? Otherwise the run will fail.", port, port + 1);
+				boolean promptToChange = wizard.promptYesNo(message);
+
+				if (promptToChange) {
+					++port;
+					wizard.showMessage(String.format("Testing port %s", port));
```

Comment:
Please remove it from here. I asked below to include a similar message in isPortInUse.